src/index.ts

```diff
@@ -138,72 +175,103 @@ export default class MondayConnector extends BaseHttpConnector<
   MondayConnectorEventOptions
 > {
   _sdk: MondaySdk
+  private webhookURL: string
+  private webhookId?: string
+  private webhookLastChangedAt?: number
 
   constructor(app: Reshuffle, options: MondayConnectorConfigOptions, id?: string) {
     super(app, options, id)
+    this.webhookURL = validateBaseURL(options.baseURL) + WEBHOOK_PATH
     this._sdk = mondaySdk({ token: options.token })
+    this.app.registerHTTPDelegate(WEBHOOK_PATH, this)
```

Comment:
I'm tempted to move this in the on function. You may want to use only the actions and so don't need to register a webhook.